modules/le_log/le_log.cpp

```diff
@@ -0,0 +1,94 @@
+#include "le_log.h"
+#include "le_core/le_core.h"
+
+#include <atomic>
+#include <mutex>
+#include <unordered_map>
+#include <cstdarg>
+
+struct le_log_module_o {
+	std::string     name      = "DEFAULT";
+	std::atomic_int log_level = 1;
+};
+
+static le_log_module_o *le_log_module_default() {
```

Comment:
It might make sense to have a logging initialization where a default context is created. but not sure where to place this startup method.

Reply:
i saw you placed it in the api registration method, which was going to be the place i was going to suggest to put it :)

Reply:
yeah, took me a second to realize that this method gets called each time with the same api object.